src/Symfony/Bundle/SecurityBundle/Resources/config/schema/security-1.0.xsd

```diff
@@ -9,11 +9,8 @@
     <xsd:complexType name="config">
         <xsd:choice maxOccurs="unbounded">
             <xsd:element name="access-decision-manager" type="access_decision_manager" minOccurs="0" maxOccurs="1" />
-            <xsd:element name="password_hashers" type="password_hashers" minOccurs="0" maxOccurs="1" />
```

Comment:
This XSD doesn't follow any existing practice (the same issue is fixed for a few other keys below).
Note that the resulting config tree is going to be exactly the same.
It's just validation that might fail if anyone uses XML to define semantic.
This will happen at compile time, so really easy to spot and fix.

Reply:
It is :) + an invitation to fix their config!